packages/genesys-spark-components/src/components/beta/gux-calendar-beta/components/single/gux-calendar.tsx

```diff
@@ -314,37 +324,22 @@ export class GuxCalendar {
             week =>
               (
                 <div class="gux-content-week">
-                  {week.dates.map(
-                    day =>
-                      (
-                        <div
-                          data-date={day.date.getTime()}
-                          onClick={() => this.onDateClick(day.date)}
-                          role="button"
+                  {week.dates.map(day => {
+                    const isoDateStr = asIsoDate(day.date);
+                    return (
+                      <slot key={isoDateStr} name={isoDateStr}>
+                        <gux-day
+                          day={isoDateStr}
                           aria-current={day.selected ? 'true' : 'false'}
-                          tabindex={day.selected || day.focused ? '0' : '-1'}
-                          onKeyDown={e => void this.onKeyDown(e)}
                           aria-disabled={day.disabled ? 'true' : 'false'}
```

Comment:
If a developer slots their own `gux-day` they will be responsible for making sure `aria-current`, `aria-disabled` and `tabindex` are correct right?